R/dependencies.R

```diff
@@ -120,14 +120,7 @@ snapshotRDependencies <- function(appDir, implicit_dependencies=c(), verbose = F
     pkg <- records[i, "Package"]
     source <- records[i, "Source"]
     repository <- NA
-    # capture Bioconcutor repository
-    if (identical(source, "Bioconductor")) {
-      if (pkg %in% biocPackages) {
-        repository <- biocPackages[pkg, 'Repository']
-      }
-    } else if (isSCMSource(source)) {
```

Comment:
What happened to the treatment of SCM sources?

Reply:
Great question: As per my underatanding previously nothing happened in `R/dependencies.R`  if `isSCMSource(source)` was true. By removing this reference altogether, SCM sources are still respected. We may possibly be able to remove the function altogether (and the reference in `R/bundle.R` as well).

As a sanity check I pushed the faithful geyser app (with shiny as of latest in github) into my local RSC and it installed the same version of shiny from github. 

```
options(repos = c(CRAN = "https://packagemanager.rstudio.com/cran/latest"))
install.packages("remotes")
remotes::install_github("rstudio/shiny")
remotes::install_github("michaelmayer2/rsconnect")
```